bases/shared/instagoric-server/server.js

```diff
@@ -82,18 +82,22 @@ const namespace =
     flag: 'r',
   });
 
-const revision =
-  process.env.AG0_MODE === 'true'
-    ? 'ag0'
-    : fs
+let revision;
+if (process.env.AGO_MODE === 'true') {
+  revision = 'ag0';
+} else {
+  try {
+    revision = fs
       .readFileSync(
         '/usr/src/agoric-sdk/packages/solo/public/git-revision.txt',
-        {
-          encoding: 'utf8',
-          flag: 'r',
-        },
+        { encoding: 'utf8', flag: 'r' },
       )
       .trim();
+  } catch (error) {
```

Comment:
I would recommend to only do this when the server is started with --fake (locally).

We would want our server to crash on start (on production k8) if this file is missing.

Reply:
Good point. Let me update

Reply:
We can even skip this file read step if it is
 --fake

Reply:
So your call

Reply:
Made the update